snews.php

```diff
@@ -2188,6 +2051,8 @@ function center() {
 // ROUTES
 addRoute('login', l('login'), 'login');
 addRoute('contact', l('contact'), 'contact');
-addRoute('sitemap', l('sitemap'), 'sitemap');
-addRoute('archive', l('archive'), 'archive');
+
+// LOAD ADDONS
+readAddons();
```

Comment:
This function won't work if call addon "hello".
We must kepp call function before function l($var)